### PR TITLE
[api-service] introduce logger, add fetching errors handling

### PR DESCRIPTION
**Pull request by junkkerrigan**
I decided on using this omegacool logging lib: [winston.js](https://github.com/winstonjs/winston).

There's a change that is not related to this PR: I updated a comment in `Aggregator` code (what I forgot to do in the api-service v0 PR).
And there's a change that is not explicitly related to logger introducing: handling of fetching errors, such as TimeoutError.

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ services:
     volumes:
       - ./package.json:/app/package.json
       - ./tsconfig.json:/app/tsconfig.json
+      - ./logs:/app/logs
       - ./src:/app/src
       - ./logs:/logs
```
Comment (xXxRisingTidexXx) [inline docker-compose-dev.yml]: А можеш, будь ласка, дропнути цей вольюм? Я його помилково втулив.

**Review by xXxRisingTidexXx — APPROVED** [review]
Все чудово, тільки переглянь останній коментар.

File: docker-compose-dev.yml (hunk 1 above)
Comment (xXxRisingTidexXx) [reply docker-compose-dev.yml]: Я про лінію #45.

Comment (junkkerrigan) [reply docker-compose-dev.yml]: eazy;)

**Pull request merged**